### PR TITLE
Advantages over SFTP

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: vipps-report-api-faq.md [hunk 1]
```diff
@@ -58,6 +60,28 @@ or on
...
 
+## What are the benefits of the Report API over the SFTP service?
+
+The
+[Vipps SFTP Report Service](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/settlements/sftp-report-service)
+lets merchants and partners download settlement files by SFTP.
+
+The Report API has some benefits over this:
+- Data can be retrieved at any time.
+- Data is available regardless of whether the balance is positive or negative,
```
Comment (dss-vipps) [inline vipps-report-api-faq.md:71]: Actually this is more of a planned feature, but there is a "bug" now that means that this isn't solved.

File: vipps-report-api-faq.md [hunk 2]
```diff
@@ -58,6 +60,28 @@ or on
 The only difference is that the data can be fetched over a more modern REST API.
 We aim to provide more information through this API in the future.
 
+## What are the benefits of the Report API over the SFTP service?
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:63]:
```suggestion
<!-- START_COMMENT -->
## What are the benefits of the Report API over the SFTP service?
```

File: vipps-report-api-faq.md [hunk 3]
```diff
@@ -58,6 +60,28 @@ or on
...
+- It is possible to retrieve data for more than one day in one request, so
+  it is possible to retrieve an entire weekend, week, month, etc. in one request.
+  The SFTP service provides one ´file per day.
+- The data is in JSON format, making it easy to use it in many different ways.
+- Later: Support for
+  [partner keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-partner/partner-keys),
+  so an accounting partner can use its API keys for all its merchants.
+  The merchant only needs to
+  [give access to an accounting partner](https://vippsas.github.io/vipps-developer-docs/docs/APIs/report-api/api-guide/overview#give-access-to-an-accounting-partner)
+  on [portal.vipps.no](https://portal.vipps.no).
```
Comment (rebekaburnett) [inline vipps-report-api-faq.md:83]:
```suggestion
  on [portal.vipps.no](https://portal.vipps.no).
  <!-- END_COMMENT -->
```

File: vipps-report-api-faq.md (hunk 1 above)
Comment (rebekaburnett) [reply vipps-report-api-faq.md:72]: We'll comment it out for now and then it won't show up. We can update it more when the bug is fixed. :-)

**Review by rebekaburnett — APPROVED** [review]
We should probably

**Comment by rebekaburnett** [comment]
I removed the TOC because we don't require them any more, we don't need them on Docusaurus, and they are annoying to keep up to date. hee hee

**Pull request merged**